### PR TITLE
Add Nick and George as maintainers

### PR DESCRIPTION
**Pull request by macfarla**
I propose to add @nicksneo and @gtebrean as Besu project maintainers. In addition to quality code contributions, they have been contributing to Besu community via contributing to discussions in Discord. 

Nick has made a number of code contributions including -
* show correct revert reason data in priv_call
* removed GoQuorum permissioning interop
* migrated several packages from Junit4 to Junit5

Nick’s PRs: https://github.com/hyperledger/besu/pulls?q=is%3Apr+is%3Aclosed+author%3ANickSneo

George has made a number of code contributions including - 
* bugfix in eth_feeHistory
* decoupled JsonRPCError enum from the data field
* made pretty JSON feature user-configurable

George's PRs: https://github.com/hyperledger/besu/pulls?q=is%3Apr+is%3Aclosed+author%3Agtebrean

Voting ends two weeks from today.

For more information on this process see the Becoming a Maintainer section in the MAINTAINERS.md file.

### CODE REVIEW

**Review by siladu — APPROVED** [review]

**Review by jframe — APPROVED** [review]

**Comment by garyschulte** [comment]
I am willing to be convinced otherwise, but since we only require one ✅ to merge a PR to main, I'd like to see examples of more substantive commits before :+1: 

Also, shouldn't we split these into two separate PRs since we are effectively voting on the addition of 2 different maintainers?

**Comment by matkt** [comment]
I completely agree with Gary. I would also like to see more contributions before giving my approval. On the second point, I also agree with Gary that we should create two separate pull requests, one for each contributor.

**Comment by jframe** [comment]
Agree that there should be a separate PR for each proposed maintainer

**Comment by macfarla** [comment]
will close this in favour of separate PRs for individual maintainers

**Pull request closed without merging**
